<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_emailList</class>
 <widget class="QDialog" name="Dialog_emailList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:gray;</string>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_selectedDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>선택된 날짜</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
color:white;
font:bold;
background-color:#3e3e3e;
}
QTableWidget QHeaderView { background-color: #00000000;
color: white;}
QTableWidget QHeaderView::section { background-color:qlineargradient(spread:reflect, x1:1, y1:0, x2:0.995, y2:1, stop:0 rgba(100,100, 100, 255), stop:0.305419 rgba(50, 50, 50, 255), stop:0.935961 rgba(10, 11, 18, 0), stop:1 rgba(50, 50, 50, 255)); border-radius:2px;}
QTableWidget QHeaderView::section:checked { background-color: #FFC000; color: #000000;}
QHeaderView::section:horizontal { border-right: 1px solid #00000000; }
QHeaderView::section:vertical { border-bottom: 1px solid #AAAAAA; }</string>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>시작</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>종료 </string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>내용</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>calendar</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_prev_month">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next_month">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_today">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{	
	font:bold;
	font-size:12pt;
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(75,75, 75, 255), stop:1 rgba(50, 50, 50, 255));
	border-radius: 10px;
	border:1px solid rgb(150,150,150);

}</string>
   </property>
   <property name="text">
    <string>TODAY</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
